java/pom.xml

```diff
@@ -642,6 +642,11 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.jetbrains</groupId>
```

Comment:
nit: required? can we use Objects.requireNotNull instead?

Reply:
I really want an annotation to mark the intent and help with static analysis. I really don't need a run time check.